web/src/components/FilingsCalendar.test.tsx

```diff
@@ -476,6 +476,51 @@ describe("<FilingsCalendar />", () => {
         screen.queryByText(Config.dashboardDefaults.calendarGridViewButton, { exact: false })
       ).not.toBeInTheDocument();
     });
+
+    it("doesn't display past months in list view", () => {
+      const pastDueDate = getCurrentDate().subtract(1, "months");
+      const pastReport = generateTaxFiling({
+        identifier: "past",
+        dueDate: pastDueDate.format(defaultDateFormat),
+      });
+
+      const futureDueDate = getCurrentDate().add(2, "months");
+      const futureReport = generateTaxFiling({
+        identifier: "future",
+        dueDate: futureDueDate.format(defaultDateFormat),
+      });
+
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: randomElementFromArray(
+            OperatingPhases.filter((obj) => {
+              return obj.displayCalendarType === "LIST" || obj.displayCalendarType === "FULL";
+            })
+          ).id,
+        }),
+        taxFilingData: generateTaxFilingData({ filings: [pastReport, futureReport] }),
+      });
+
+      const operateReferences: Record<string, OperateReference> = {
+        past: {
+          name: "Past Report",
+          urlSlug: "past-report-url",
+          urlPath: "past_report-url-path",
```

Comment:
if these values aren't being used in the test, we should use a generator instead of writing them out explicitly. That way, it's clear to a reader of the test that it's not important/relevant